net/tor/src/opnsense/mvc/app/models/OPNsense/Tor/General.xml

```diff
@@ -26,9 +26,11 @@
     </control_port>
     <control_port_password type="TextField">
       <Required>N</Required>
+      <mask>/^.+$/</mask>
```

Comment:
that just means it can't be empty, coupled with Required=N wouldn't that prevent resetting the value to an empty string which is allowed?

Reply:
control port password cannot be reset - at least I don't provide this option to the UI.
This is an automatically generated key.

Reply:
See: https://github.com/fabianfrz/plugins/blob/21453ff9b2267ba63d945a3ee3c4df70dbb20c5f/net/tor/src/opnsense/mvc/app/controllers/OPNsense/Tor/Api/GeneralController.php#L63

Reply:
if it's not in the UI ok :)